unopim/unopim
# Elastic search

## Issue Reference
Elastic search was not working for products table.

## Description
Implemented Elastic search ^8.17. 

## How To Test This?
Start new project
Add values in Elastic search variables in .env
Run the following command for indexing
~~~
php artisan indexer:index
~~~

Run the following command for check the data count in indexes
~~~
GET "http://localhost:9200/_cat/shards?v"
~~~
## Documentation
[packages/Webkul/ElasticSearch/Guide.md](https://github.com/kartikeywebkul9260/unopim-fork/blob/ElasticSearch/packages/Webkul/ElasticSearch/Guide.md)

## Branch Selection
['master']
